src/library/playlistfeature.h

```diff
@@ -1,6 +1,3 @@
-// playlistfeature.h
```

Comment:
If your remove those comments please also replace the legacy include guards with `#pragma once`

Reply:
this is the sort of thing that should just be done for all files at once with a commit to master tbh ^^^

Reply:
Mass code style changes cause unnecessary merge conflicts with open pull requests. We update style as we work instead.

Reply:
In general I agree, but how would replacing these lines create a conflict?

Reply:
Please let's stop discussing minor details and get things done. We have to agree on certain processes.

Reply:
Still waiting for the #pragma once change. Please don't resolve conversations if you don't follow the request.

Reply:
Sorry, I had done it on all files except the one you commented on :see_no_evil: